Sources/_FoundationInternals/String/String+Internals.swift

```diff
@@ -54,3 +54,12 @@ extension String {
     }
 
 }
+
+extension DefaultStringInterpolation {
+    mutating func appendInterpolation(_pad value: Int, _toWidth width: Int, _using paddingString: String) {
+        for i in 0...(String(value).count - 1) {
+            appendLiteral(paddingString)
+        }
+        appendInterpolation(value)
+    }
```

Comment:
```suggestion
    mutating func appendInterpolation(_zeroPad value: Int, _toWidth width: Int) {
      precondition(width > 0)
      let representation = String(value)
      let count = width &- representation.utf8.count
      if count > 0 {
        appendLiteral(String(repeating: "0", count: count))
      }
      appendLiteral(representation)
    }
```
For a slightly simpler call site. If we really want to have the padding as a parameter, it should be a `Character`.